<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiffConcepts</class>
 <widget class="QWidget" name="DiffConcepts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>953</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="continue1">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>10</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Далее</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>1)</string>
   </property>
  </widget>
  <widget class="QLabel" name="concept1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Заключение</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>2)</string>
   </property>
  </widget>
  <widget class="QLabel" name="concept2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Заключение</string>
   </property>
  </widget>
  <widget class="QLabel" name="concept3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Заключение</string>
   </property>
  </widget>
  <widget class="QComboBox" name="pos_pole">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>700</width>
     <height>25</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="duplicatesEnabled">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>381</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Критерий:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>3)</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="conclusionEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>700</width>
     <height>58</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>78</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(225, 225, 225)</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string>Выберите из списка или введите свой критерий, по которому первое заключение отличается от двух других, или нажмите &quot;Далее&quot;,  если такого критерия нет.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
